projects/wave-core-new/src/lib/backend/backend.model.ts

```diff
@@ -519,6 +519,7 @@ export interface LayerCollectionItemDict {
     id: ProviderLayerIdDict | ProviderLayerCollectionIdDict;
     name: string;
     description: string;
+    properties?: [string, string][];
 }
 
```

Comment:
You forgot to refactor the data atlas. It still tries to use the old provider.

Reply:
should I re-add the ebv handlers then? The task was to migrate the EBV analyzer and not the data atlas.

Reply:
Works quite well. Just please bring back the card for each dataset and provide within that card your hierarchical selection. Then it's fine I guess.

Reply:
so on loading of the application we want to download three levels of layer hierarchy? This will mean a couple dozen API requests just for showing the layers. Maybe we should rethink the layer collection API if that's what we want. Especially as the requests depend on each other. We first have to load the top level and then the datasets and then for each dataset the scenarios. I guess we need an API call that gives us the hierarchy up to a certain point all at once then. What do you think?

Reply:
We could think of this. On the other hand, you could only load this if the accordion get's opened.

Maybe, we collect that into potential improvements. I guess you don't want to implement it now?

Reply:
642ff1f5e751e297cf375c6cb23e5af6f6a10185